### DISCUSSION TITLE
[.NET7] Failing to setup UNO

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#10446] [Q&A] [answered]
I'm going through instructions [here ](https://platform.uno/docs/articles/get-started-dotnet-new.html?tabs=windows) and am running 
`uno-check --pre`

At some point it says I would need Python in order to build WebAssembly AOT (see 3rd screenshot), and it directs me to the MS store page. When pushing the "Get in app store" button I only gives me the window as per first screenshot ... which makes me believe I migth already have installed Python ... which I confirmed as per 2nd screenshot.

Why would uno-check complain about a missing Python installation and how could I rectify the issue?

<img width="375" alt="image" src="https://user-images.githubusercontent.com/26179242/201980546-1cebb775-01c4-4c27-9ed4-670052c1c46b.png">

<img width="752" alt="image" src="https://user-images.githubusercontent.com/26179242/201980740-ad0b8711-4f27-4433-92e6-55e1ffb442b7.png">

<img width="675" alt="image" src="https://user-images.githubusercontent.com/26179242/201981059-65f4da77-2f91-4229-b266-b59b8d53c8a9.png">

**Comment by jeromelaban** [comment, marked as answer]
We have an opened issue (https://github.com/unoplatform/uno.check/issues/107) about it, it's not fixed yet. You can skip this check.

**Reply by DierkDroth** [reply]
Thanks for your feedback @jeromelaban. Note, I'm not on an ARM64 machine (as the linked issue suggests)
